### PR TITLE
Reduce stale time

### PR DESCRIPTION
**Pull request by VaibhavSys**
I think that the stale time is too much and reducing it is a good idea

### CODE REVIEW

**Review by DEV-DIBSTER — DISMISSED** [review]

**Comment by phenax** [comment]
Entirely up to you. @is-a-dev/maintainers, thoughts?

**Comment by DEV-DIBSTER** [comment]
Agreed, I want to test this out, and I might even say make it a week for stale.

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -13,8 +13,8 @@ jobs:
     - uses: actions/stale@v3
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        days-before-stale: 30
-        days-before-close: 20
+        days-before-stale: 10
```
Comment (WilliamDavidHarrison) [inline .github/workflows/stale.yml:16]:
```suggestion
        days-before-stale: 7
```

Comment (WilliamDavidHarrison) [reply .github/workflows/stale.yml:16]: I feel like this would be better.

**Review by DEV-DIBSTER — APPROVED** [review]

**Review by wdhdev — APPROVED** [review]

**Review by mtgsquad — APPROVED** [review]

**Comment by WilliamDavidHarrison** [comment]
@phenax Can you merge?

**Review by andrewstech — APPROVED** [review]

**Review by phenax — APPROVED** [review]

**Pull request merged**